assets/src/styles/blocks/core-overrides/Table.scss

```diff
@@ -4,44 +4,44 @@
   table {
     // Grey background (defaut)
     th {
-      background-color: $table-header-grey;
+      background-color: var(--table-header-grey--background, $table-header-grey);
     }
 
     tr {
-      background-color: $table-even-row-grey;
+      background-color: var(--table-even-row-grey--background, $table-even-row-grey);
     }
 
     tfoot td {
-      background-color: $table-footer-grey;
+      background-color: var(--table-footer-grey--background, $table-footer-grey);
```

Comment:
This is more of a design question than a dev question, so maybe something to check with Houssam, I find it weird that the footer color is the same as some rows if we have stripes, because they might end up next to each other:
<img width="713" alt="Screenshot 2023-04-07 at 09 23 05" src="https://user-images.githubusercontent.com/6949075/230562460-3d1be3b8-26fe-4913-a74e-eab4c2833e85.png">

Reply:
Houssam suggested to make the text for the Footer bold.

Reply:
Well ignore my last comment 😅.
After talking with him again, we figured out some color changes, pushing those now.